proxy/server/registry_events.go

```diff
@@ -0,0 +1,39 @@
+package server
+
+import "time"
+
+// Notification holds all events.
+type Notification struct {
+	Events []Event
+}
+
+// Event holds the details of a event.
+type Event struct {
```

Comment:
Seems like a lot of these fields are unused. Can we:
1. Remove unused fields / structs to keep things succinct (this will make testing much easier).
2. Link to the original notifications API in a comment in case we need to add some of these back in.

Reply:
ok